ignite/metrics/metrics_lambda.py

```diff
@@ -0,0 +1,55 @@
+from ignite.metrics.metric import Metric
+
+
+class MetricsLambda(Metric):
+    """
+    Apply a function to other metrics to obtain a new metric.
+    The result of the new metric is defined to be the result
+    of applying the function to the result of argument metrics.
+
+    When update, this metric does not recursively update the metrics
+    it depends on. When reset, all its dependency metrics would be
+    resetted. When attach, all its dependencies would be automatically
+    attached.
+
+    Arguments:
+        f (callable): the function that defines the computation
+        args (sequence): Sequence of other metrics or something
+            else that will be fed to ``f`` as arguments.
+
+    Examples:
+        >>> precision = Precision()
+        >>> recall = Recall()
+        >>> def Fbeta(r, p, beta):
+        >>>     return (1 + beta ** 2) * p * r / (beta ** 2 * p + r)
```

Comment:
```suggestion
        >>>     return torch.mean((1 + beta ** 2) * p * r / (beta ** 2 * p + r)).item()
```

Reply:
@vfdev-5 Shall we update the unit test accordingly?

Reply:
It is the case, not ?